.github/workflows/lint.yaml

```diff
@@ -1,6 +1,12 @@
 name: lint
 on:
-  [push, pull_request]
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
Why do we need to run the lint workflow once per week?

Reply:
No matter the linter that we eventually settle on, it is an external dependency and I think we would want to know if an external change influences the local linter behaviour.
An example where this is relevant would be a PR failing due to the linter, where the problem lies not in the code touched by the PR, but due to more stringent testing following an external code change.

But that's just an opinion, if you don't agree we can remove the regular builds here.

Reply:
Ah sure, that's a valid enough reason.